### PR TITLE
Moved MAILQ to optional CONSTANT

### PR DESCRIPTION
**Pull request by stefaweb**

### CODE REVIEW

File: mailscanner/functions.php [hunk 1]
```diff
@@ -455,7 +455,7 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
                 }
 
                 // Else use MAILQ from conf.php which is for Sendmail or Exim
-            } elseif (MAILQ && !DISTRIBUTED_SETUP) {
+            } elseif (defined(MAILQ) && !DISTRIBUTED_SETUP) {
```
Comment (endelwar) [inline mailscanner/functions.php]: a check that MAILQ is true is missing

**Comment by stefaweb** [comment]
ok, PR rebased

**Comment by Skywalker-11** [comment]
still merge tags in conf.example

**Comment by stefaweb** [comment]
> still merge tags in conf.example

done

**Comment by stefaweb** [comment]
Can be merged.

File: mailscanner/functions.php [hunk 2]
```diff
@@ -4141,6 +4140,7 @@ function checkConfVariables()
         'EXIM_QUEUE_OUT' => array('description' => 'needed only if using Exim as MTA'),
         'PWD_RESET_FROM_NAME' => array('description' => 'needed if Password Reset feature is enabled'),
         'PWD_RESET_FROM_ADDRESS' => array('description' => 'needed if Password Reset feature is enabled'),
+        'MAILQ' => array('description' => 'needed to display the inbound/outbound mail queue lengths'),
```
Comment (Skywalker-11) [inline mailscanner/functions.php]: It is not needed for postfix to show queue length

**Pull request merged**